tasks.py

```diff
@@ -238,6 +238,18 @@ def postgresql(c):
     c.run('sudo apt-get install postgresql postgresql-contrib libpq-dev')
     c.run('pip3 install psycopg2')
 
+@task
+def rabbitmq(c):
```

Comment:
My preference is for Redis, as it can also be used for cache too, but it does need a separate install. Does RabbitMQ broker needs to be installed separately? Does it supports caching?

Reply:
I can add a target for Redis also.  Most linux distros and BSDs have native packages available for them.  Currently it's heavily skewed to apt based linux distros.

If you want a single solution for a small combined cache/broker then Redis is the way to go, you can always have both Redis and RabbitMQ installed for a large environment.